<launch>
  <!-- Read arguments -->
  <arg name="port" default="/dev/ttyUSB0"/>
  <arg name="frame" default="sonar"/>
  <!-- Launch user interface -->
  <node name="rpim"
        pkg="stams_gui"
        type="rpim.py"
        output="screen"
        respawn="false">
  </node>

      <!-- Launch Profiling Sonar -->
  <node name="tritech_profiler"
        pkg="tritech_profiler"
        type="scan.py"
        output="screen"
        respawn="true">
    <!-- Set ROS parameters -->
    <param name="port" value="$(arg port)"/>
    <param name="frame" value="$(arg frame)"/>
  </node>

  <!-- Launch Valeport Altimeter -->
  <node name="valeport_altimeter"
        pkg="valeport_altimeter"
        type="dynamic_altimeter.py"
        output="screen"
        respawn="true">
  </node>

  <!-- Launch Depth info board -->
  <node name="depth_driver"
        pkg="stams_gui"
        type="depthstamp.py"
        output="screen"
        respawn="true">
  </node>

  <!-- Launch camera -->
  <!-- Set this to your camera's name -->
  <arg name="cam_name" value="bowtech_camera" />
  <arg name="DEVICE" default="/dev/video1"/>


      <!-- Launch rqt_reconfigure Sonar -->
  <node name="rqt_reconfigure"
        pkg="rqt_reconfigure"
        type="rqt_reconfigure"
        output="screen"
        >
  </node>


</launch>
